static/app/utils/discover/fields.tsx

```diff
@@ -262,8 +263,13 @@ export const AGGREGATIONS = {
     multiPlotType: 'line',
   },
   apdex: {
-    generateDefaultValue({parameter, organization}: DefaultValueInputs) {
-      return organization.apdexThreshold?.toString() ?? parameter.defaultValue;
+    getFieldOverrides({parameter, organization}: DefaultValueInputs) {
+      return organization.features.includes('project-transaction-threshold')
+        ? {required: false, placeholder: 'Automatic', defaultValue: ''}
+        : {
+            defaultValue:
+              organization.apdexThreshold?.toString() ?? parameter.defaultValue,
+          };
```

Comment:
Nit: it might be easier to write this with just `if` statements since this ternary is getting quite long.